### PR TITLE
Development

### PR DESCRIPTION
**Pull request by SunnyAsar**
please review our project.
youtube page with HTML and CSS, embedded video and side menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+       <ul class="right">
+           <li><i class="fas fa-video"></i></li>
+           <li><i class="fas fa-th"></i></li>
+           <li><i class="fas fa-comment"></i></li>
+           <li><i class="fas fa-bell"></i></li>
+       </ul>
+    </nav>
+    <main class="main-section">
+        <div class="media">
+            <div class="video" style="position:relative;padding-top:56.25%;">
```
Comment (zlayabekrija) [inline index.html]: Please avoid using inline styling. Move your styles to css instead.
Padding of 56 % is huge, please use another approach.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+<body>
+    <nav class="navbar" id="navbar">
+        <ul class="left">
+            <li><i class="fas fa-bars icon"></i></li>
+            <li><i class="fab fa-youtube youtube-icon" ></i>Youtube</li>
+        </ul>
+       <div class="search">
+           <input type="text" name="" id="" placeholder="Search for Video">
```
Comment (zlayabekrija) [inline index.html]: id and name shouldn't be empty or shouldn't be there.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+           <li><i class="fas fa-video"></i></li>
+           <li><i class="fas fa-th"></i></li>
+           <li><i class="fas fa-comment"></i></li>
+           <li><i class="fas fa-bell"></i></li>
+       </ul>
+    </nav>
+    <main class="main-section">
+        <div class="media">
+            <div class="video" style="position:relative;padding-top:56.25%;">
+                <iframe width="100%" height="100%" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="position:absolute;top:0;left:0;width:100%;height:100%;"></iframe>
```
Comment (zlayabekrija) [inline index.html]: Width and height should be in pixels,frameborder is obsolute.
Again no inline styling please. Position absolute isn't necessary, you can use a different way to center your content.

**Review by zlayabekrija — COMMENTED** [review]
Great work guys,only minor changes and you are ready to go.
Please follow up with these changes.
Please use https://htmlpreview.github.io/ ,for your online preview on the page.
Path of css file isn't proper, please fix it.
After making these changes, leave the comment, I'll review it and then you can proceed with the next projects.Chrome is ok but Firefox not.

Also please use https://validator.w3.org/ for your code validation.

File: index.html (hunk 3 above)
Comment (HenryTabima) [reply index.html]: Thanks a lot for the review, we're going to make the changes

**Review by zlayabekrija — APPROVED** [review]
Thumbs up 👍 . Great work, you can merge this pull request.
Happy coding.

**Comment by HenryTabima** [comment]
@zlayabekrija  thank you for reviewing our code!

**Pull request merged**